modules/common/deps.sh

```diff
@@ -51,8 +51,17 @@ else
         info "CF cli version:"
         cf version
     fi
-fi
 
+    if [ ! -e "bin/eksctl" ]; then
```

Comment:
do not forget Dockerfile

Reply:
There is no dockerfile in the eksctl tarball.
... :bulb: You are talking about a Dockerfile in catapult ?!
... I see, top dir `Dockerfile`.
Will do.

Reply:
Done with commit c03441fd40b05f9dce399c1e71f9bfed43b080cb